packages/core/src/components/DialogBox/DialogBox.stories.mdx

```diff
@@ -22,25 +22,32 @@ DialogBox consists of three sections Header, Content and Actions. All of these y
 </DialogBox>
 ```
 
-Shadow will appear on the Header & Actions section if you scroll the content.
-To close the dialog-box you can perform below actions:
+Shadow appears on the Header and Actions section if you scroll through the content.
 
-1. Click on the cross icon
-2. Press `Esc` key
-3. If you have provided `shouldCloseOnOutsideClick` this prop then dialog-box will close on clicking outside of dialog-box
+To close the dialog box, follow the steps given below:
+
+1. Click the cross icon.
+2. Press `Esc` key.
+3. If the `shouldCloseOnOutsideClick` prop is selected, then dialog box will close on clicking outside the dialog box.
```

Comment:
I think `selected` is not the right word over here, we can use `provided` or `set true`